src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,44 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<E,T> {
+    private E first;
+    private T second;
 
+    private Pair(E first, T second) {
+        this.first = first;
+        this.second = second;
+    }
+
+    public E getFirst() {
+        return first;
+    }
+
+    public T getSecond() {
+        return second;
+    }
+
+    @Override
+    public boolean equals(Object element) {
+        if (this == element) {
+            return true;
+        }
+        if (element == null || getClass() != element.getClass()) {
+            return false;
+        }
+        Pair<?, ?> pair = (Pair<?, ?>) element;
```

Comment:
instead of '?' may be anything. But we should have the current K and V parameter

Reply:
You may ignore idea warnings

Reply:
done